### DISCUSSION TITLE
Thanks

### DISCUSSION THREAD

**Discussion by Drallas** [tteck/Proxmox#1978] [Show and tell]
I'm using many of your scripts, such a time saver and convenience! 

<img width="1300" alt="Screenshot 2023-10-29 at 11 32 40" src="https://github.com/tteck/Proxmox/assets/24792888/dc84e48f-e01f-4038-825c-e10ad0e51e80">

**Comment by di5cord20** [comment]
@Drallas - i'm a proxmox noob; I like the table view in your screenshot but can't find the equivalent in my setup.  Am I missing something?  Was this a custom view you built?

**Comment by di5cord20** [comment]
Disregard - just found it :).  For anyone else who wants the same view, change "Server view" to "Folder view" and then click the "LXC Container" parent folder.
